src/explore-education-statistics-admin/src/pages/admin-dashboard/components/MethodologySummary.tsx

```diff
@@ -96,118 +96,127 @@ const MethodologySummary = ({
-                        className="govuk-button--secondary govuk-!-margin-left-4"
-                      >
-                        View original methodology
-                      </ButtonLink>
-                    </ButtonGroup>
-                  </>
-                ) : (
-                  <>
-                    <ButtonGroup>
-                      <ButtonLink
-                        to={generatePath(methodologySummaryRoute.path, {
-                          publicationId,
-                          methodologyId: methodology.id,
-                        })}
-                      >
-                        {methodology.permissions.canUpdateMethodology
-                          ? 'Edit this methodology'
-                          : 'View this methodology'}
-                      </ButtonLink>
-                      {methodology.permissions
-                        .canMakeAmendmentOfMethodology && (
-                        <Button
-                          type="button"
-                          onClick={() => setAmendMethodologyId(methodology.id)}
-                          variant="secondary"
+              <div className="govuk-grid-row">
+                <div className="govuk-grid-column-two-thirds">
+                  {methodology.amendment ? (
+                    <>
+                      <ButtonGroup>
+                        <ButtonLink
+                          to={generatePath(methodologySummaryRoute.path, {
+                            publicationId,
+                            methodologyId: methodology.id,
+                          })}
                         >
-                          Amend methodology
-                        </Button>
-                      )}
-                    </ButtonGroup>
-                  </>
-                )}
-              </div>
-              <div className="govuk-grid-column-one-third dfe-align--right">
-                {methodology.permissions.canDeleteMethodology && (
-                  <Button
-                    onClick={() =>
-                      setDeleteMethodologyDetails({
-                        methodologyId: methodology.id,
-                        amendment: methodology.amendment,
-                      })
-                    }
-                    className="govuk-button--warning"
-                  >
-                    {methodology.amendment ? 'Cancel amendment' : 'Remove'}
-                  </Button>
-                )}
+                          {canEdit
+                            ? 'Edit this amendment'
+                            : 'View this amendment'}
+                        </ButtonLink>
+                        <ButtonLink
+                          to={generatePath(methodologySummaryRoute.path, {
+                            publicationId,
+                            methodologyId: methodology.previousVersionId,
+                          })}
+                          className="govuk-button--secondary govuk-!-margin-left-4"
```

Comment:
Could use `variant="secondary"` prop instead of the `govuk-button--secondary` class